src/pkg/bundle/deploy.go

```diff
@@ -358,11 +357,9 @@ func (b *Bundle) ConfirmBundleDeploy() (confirm bool) {
 	pkgviews := formPkgViews(b)
 
 	message.HeaderInfof("🎁 BUNDLE DEFINITION")
-	pterm.Println("kind: UDS Bundle")
-
-	message.HorizontalRule()
 
 	message.Title("Metatdata:", "information about this bundle")
```

Comment:
Still seeing `Metatdata` vs `Metadata`, but your screenshot is correct?

Reply:
Whoops, I think that got restored when I rebased earlier. I fixed it (again) and removed the `kind:` part in 0a57611

Reply:
I also updated the screenshot included to reflect the latest